python3-pyqt5.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "python3-modules",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "pyqt5",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-deps PyQt5-5.12.3-5.12.10-cp35.cp36.cp37.cp38.cp39-abi3-manylinux1_x86_64.whl"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3b/b2/2c8492ce82de20dd241d1166214764791775fbbc59e74e8449a2914361a6/PyQt5-5.12.3-5.12.10-cp35.cp36.cp37.cp38.cp39-abi3-manylinux1_x86_64.whl",
+                    "sha256": "54b02a18d95f4c6116aafc13f9d61da5ccbe912c5e08a8afde5a81468b9ff546"
+                }
+            ]
+        },
+        {
+            "name": "sip",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-deps PyQt5_sip-12.7.2-cp38-cp38-manylinux1_x86_64.whl"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/0f/b4/00182841eebe8c2cba5859348823d9729ad7cdcea9820f3153852c1f76ae/PyQt5_sip-12.7.2-cp38-cp38-manylinux1_x86_64.whl",
```

Comment:
so your build won't work on aarch64.

You should build from source.

Reply:
also it fails to build because this binary wheel is for python 3.8. This is python 3.9